src/test/java/HW20.java

```diff
@@ -34,20 +34,12 @@ public void DeletePlayList(){
         WebElement newPlaylist; //"//a[text()='Tatsiana'"
         newPlaylist = driver.findElement(By.cssSelector("a[href='#!/playlist/36447']"));
         wait.until(ExpectedConditions.visibilityOf(newPlaylist));
-        //Actions action = new Actions(driver);
-        //action.contextClick().click(newPlaylist);
         newPlaylist.click();
 
         WebElement lastDelete;
         lastDelete =wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//button[@title='Delete this playlist']")));
-                //driver.findElement(By.cssSelector("//li[@data-testid='playlist-context-menu-delete-36447']"));
-
         lastDelete.click();
-
-
-
     }
-
     }
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)

Reply:
> Super!Thank you!

I'm glad you found it helpful.